cypress/support/pages/home.pageObject.js

```diff
@@ -7,10 +7,51 @@ class HomePageObject extends PageObject {
     return cy.getByDataCy('username-link');
   }
 
+  get authorName() {
+    return cy.getByDataCy('author-name-home-page');
```

Comment:
Ensure that the `getByDataCy` command correctly returns the element with the data-cy attribute 'author-name-home-page'.